packages/swapper/src/swappers/CetusSwapper/endpoints.ts

```diff
@@ -1,3 +1,4 @@
+import { Transaction } from '@cetusprotocol/aggregator-sdk/node_modules/@mysten/sui/transactions'
```

Comment:
Didn't find any other possibilities than doing that as we need types from the aggregator versions...

Reply:
Can use this guy instead
```typescript
import { Transaction } from '@mysten/sui/dist/cjs/transactions'
```

<img width="1478" height="589" alt="Image" src="https://github.com/user-attachments/assets/cc15bfec-65db-470e-9a36-ca8d7b969c6d" />